.github/pull_request_template.md

```diff
@@ -3,11 +3,12 @@ Thanks for contributing to the Amplitude JavaScript SDK! 🎉
 
 Please fill out the following sections to help us quickly review your pull request.
 --->
+
 ### Summary
 
 <!-- What does the PR do? -->
 
 ### Checklist
 
-* [ ] Does your PR title have the correct [title format](../CONTRIBUTING.md#pr-commit-title-conventions)
-* [ ] Does your PR have a breaking change?
+* [ ] Does your PR title have the correct [title format](../CONTRIBUTING.md#pr-commit-title-conventions)?
+* [ ] Does your PR have a breaking change? <!-- Yes or no -->
```

Comment:
This when I saw it in a real PR feels a little odd to be put in a checklist of things to make sure to do. Maybe word it like `Does your PR have a breaking changes tag if it has one`?